File: DXCAir-API/DCXAir-API/DCXAir-API/Models/FlightFilterDTO.cs
namespace DCXAir_API.Models
{
    /// <summary>
    ///  Clase que determina los filtros para buscar los vuelos
    /// </summary>
    public class FlightFilterDTO
    {
        //Determina el origen requerido para el vuelo
        public string Origin { get; set; }

        //Determina el destino requerido para el vuelo
        public string Destination { get; set; }

        //Determina el tipo de moneda en el que se requiere conocer el precio del vuelo
        public string CoinToConvert { get; set; }
    }
}

File: DXCAir-web/proyecto-dxcair-web/src/app/DTOs/flight.filter.dto.ts
/**
 * Clase que determina eñ dto que almacena los filtros 
 * de los vuelos a consultar
 */
export class FlightFilterDTO {
    //Determina el origen del vuelo a consultar
    public origin:string;

    //Determina el destino del vuelo
    public destination: string;

    //Determina la moneda en la que se desea consultar
    public coinToConvert: string;
}